phpBB/phpbb/di/container_builder.php

```diff
@@ -121,6 +116,11 @@ class container_builder
 	 */
 	private $env_parameters = [];
 
+	/**
+	 * @var \phpbb\db\driver\driver_interface
+	 */
+	protected $dbal_connection = null;
```

Comment:
This is somehow needed for tests to pass which I don't quite understand. It seems like our tests are wrong somehow, as this variable is only set in `inject_dbal_driver()`, but I would assume that should only be called once per container creation. @Nicofuma anything you can add here?